packages/zero-cache/src/services/view-syncer/cvr.pg-test.ts

```diff
@@ -2621,6 +2675,16 @@ describe('view-syncer/cvr', () => {
       ),
     ).toMatchInlineSnapshot(`
       [
+        {
```

Comment:
@darkgnotic - idk why this would have moved with this change and if that matters.

Reply:
it seems more correct that `stateVersion 19z` is now output before `stateVersion 1aa`